eyelid-match-ops/src/primitives/poly/test/inv.rs

```diff
@@ -0,0 +1,29 @@
+//! Tests for polynomial inverse.
+
+use super::super::*;
+use rand::Rng;
+
+/// This sampling is similar to what will be necessary for YASHE KeyGen
+/// TODO: generate Gaussian distribution instead of "uniform"
+pub fn sample() -> Poly {
+    let mut rng = rand::thread_rng();
+    let mut res = Poly::zero();
+    let max_coeff = 8;
+    let t = 2;
```

Comment:
Should these parameters be named constants or function arguments?

Reply:
I just wanted to restrict the size of coefficients, because it influences probability of finding the inverse (smaller is better, i.e. more likely to find it). For the final version of this sampling, it would be good to have named constants, but they will change a bit, since the distribution will be Gaussian instead of Uniform. And maybe some parameter will be not a constant, but an attribute of the scheme (dynamic), like for example `t`.

Reply:
Do you think this is a blocker for merging this PR, or can it go in a ticket?

Reply:
No, this is not a blocker. The sampling will be important during the implementation of YASHE, and we have a ticket for it. Will describe those sampling methods in this ticket. 

For the inverse, the only thing we need is to increase the likelihood of finding an invertible element, so this is good enough.